### PR TITLE
[DFFE-7] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by kaozarat16**
phone product page 1
phone product page 2
tab product page 1
tab product page 2
productpage

### CODE REVIEW

File: submissions/kaozaratAmoo/html/Product.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                       </div>
+                       <div class="connect">
+                          <p>CONNECT WITH US</p>
+                           <p class="social"><a href="#"><image src = "images/facebook.png" width="60" height="40"></image></a>
+                               <a href="#"><image src = "images/twitter.png"  width="60" height="40"></image></a>
+                               <a href="#"> <image src = "images/email.png"  width="60" height="40"></image></a> 
+                               <a href="#"><image src = "images/youtube.png"  width="60" height="40"></image></a></p>
+                       </div>
+                </div>
+                               <P class ="copy"> Copyright 2020 Agrobay.com.
```
Comment (Ireti203) [inline submissions/kaozaratAmoo/html/Product.html:82]: This should be p and not P

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Remove spaces from image names

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/kaozaratAmoo/html/Product.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                               <a href="#"><image src = "images/twitter.png"  width="60" height="40"></image></a>
+                               <a href="#"> <image src = "images/email.png"  width="60" height="40"></image></a> 
+                               <a href="#"><image src = "images/youtube.png"  width="60" height="40"></image></a></p>
+                       </div>
+                </div>
+                               <P class ="copy"> Copyright 2020 Agrobay.com. 
+                                   <span class="rights">All rights reserved.</span></p> 
+                                   
+                       </div>
+                       </footer>
```
Comment (adaugochi) [inline submissions/kaozaratAmoo/html/Product.html:86]: No closing `</body>` tag

File: submissions/kaozaratAmoo/html/Product.html (hunk 1 above)
Comment (adaugochi) [reply submissions/kaozaratAmoo/html/Product.html:82]: Also fix this comment by Ireti

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove spaces on images names. Replace with `-` or `_`

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**